portal/auth.py

```diff
@@ -22,7 +22,9 @@ def login():
 
     if user is None:
         error = 'Incorrect email.'
-    elif not user['password']:
+        print("Email")
+    elif not check_password_hash(user['password'], password):
+        print("Password")
```

Comment:
Small thing, but try not to leave debugging print lines in your final PR for production. You don't want to clutter the logs and definitely don't want to accidentally leak sensitive info.